torch/onnx/symbolic_opset11.py

```diff
@@ -221,19 +221,19 @@ def __interpolate(g, input, size, scale_factor, mode, align_corners, recompute_s
     return sym_help.__interpolate_helper(g, input, size, scale_factor, mode, align_corners, recompute_scale_factor)
 
 @parse_args("v", "i", "v", "v")
-def gather(g, self, dim, index, sparse_grad=False):
+def gather(g, self, dim, index, sparse_grad=False, unique_indices=False):
     if sym_help._maybe_get_const(sparse_grad, "i"):
         return _unimplemented("gather", "sparse_grad == True")
```

Comment:
I'm not sure whether or not I should add `_unimplemented()` calls for `unique_indices == True`. I'm not very familiar with the ONNX stuff

Reply:
I decided to just add the `_unimplemented()` calls. I can remove them if needed